packages/android_intent_plus/android/src/main/java/dev/fluttercommunity/plus/androidintent/IntentSender.java

```diff
@@ -98,6 +98,9 @@ boolean canResolveActivity(Intent intent) {
     }
 
     final PackageManager packageManager = applicationContext.getPackageManager();
+    boolean result =
+        packageManager.resolveActivity(intent, PackageManager.MATCH_DEFAULT_ONLY) != null;
+    Log.d(TAG, "" + result);
```

Comment:
I'd prefer to remove this log if possible

Reply:
Yes, it will be removed in next commit